CODEOWNERS

```diff
@@ -93,8 +93,11 @@
 # The `minio` subchart
 /resources/core/charts/minio/ @akucharska @michal-hudy @pkosiec @mjasinski5 @aerfio @magicmatatjahu
 
-# The `event-bus` subchart
-/resources/core/charts/event-bus/ @a-thaler @montaro @abbi-gaurav @marcobebway @radufa @suleymanakbas91
+# The `event-bus` chart
+/resources/charts/event-bus/ @a-thaler @montaro @abbi-gaurav @marcobebway @radufa @suleymanakbas91
```

Comment:
Should this be **/resources/event-bus/**?

Reply:
fixed